app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :user, foreign_key: :user_id
```

Comment:
- [ ] `user` should have been modified to the `author` as in the ERD. Let's kindly fix it and tell Rails the class name you did the modification by updating this line of code to: `belongs_to :author, class_name: 'User'`

Reply:
The comments table has a `userId` not `authorId` should I still do this? I am actually really confused, I would really appreciate if you could clarify your point of view.

Reply:
@Muhammad0602 please check the ERD, it should be authorId 
![Screenshot from 2023-06-15 10-32-00](https://github.com/Muhammad0602/Blog-app/assets/63087491/199eae4c-fd8e-44ba-89b3-91555a076b0a)